transforms/helpers/parse-helper.ts

```diff
@@ -202,7 +201,7 @@ interface EOCallExpressionProps {
 /**
  * Parse ember object call expression, returns EmberObjectExpression and list of mixins
  */
-export function parseEmberObjectExtendExpression(
+export function parseEOExtendExpression(
```

Comment:
I usually hate reduce, but this seems like a reasonable place for it?

```suggestion
  return eoExtendExpression.arguments.reduce<EOCallExpressionProps>(
    ({ eoExpression, mixins }, arg) => {
      if (isEOExpression(arg)) {
        return { eoExpression: arg, mixins };
      } else {
        return { eoExpression, mixins: [...mixins, arg] };
      }
    },
    { eoExpression: null, mixins: [] }
  );
```